deployment/stack.yaml

```diff
@@ -0,0 +1,48 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Resources:
+  MiraiApplication:
+    Type: AWS::ElasticBeanstalk::Application
+    Properties:
```

Comment:
Add the following under Properties next to Description:
`ApplicationName: mirai-app`